tinygrad/runtime/ops_metal.py

```diff
@@ -84,15 +92,42 @@ def copyin(self, dest:Any, src:memoryview): self.as_buffer(dest)[:] = src
   def copyout(self, dest:memoryview, src:Any): dest[:] = self.as_buffer(src)
 
 class MetalDevice(Compiled):
+  captureManager = None
+
   def __init__(self, device:str):
     self.device = Metal.MTLCreateSystemDefaultDevice()
+    self.dname = device
+
+    if MTL_CAPTURE_ENABLED and not MetalDevice.captureManager:
```

Comment:
Want to make this a startCapture method to match? Also, why is stopCapture static? It looks like it depends on the device

Reply:
It depends on MTLDevice but there is only one of them (`Metal.MTLCreateSystemDefaultDevice()` always returns the same thing)

It is static because there is multiple `MetalDevice` instances created but only one `captureManager` is needed.

Reply:
Hmm, I see, because the multiple METAL devices are really a hack. Okay, static is fine, but that should be a method